optima/scenarios.py

```diff
@@ -130,7 +130,8 @@ def makescenarios(project=None, scenlist=None, verbose=2):
                     else: 
                         errormsg = 'Unrecognized population or partnership type.'
                         raise OptimaException(errormsg)
-                    for pop in pops:
+
+                    for popno, pop in enumerate(pops):
```

Comment:
personally i prefer length of variable name to roughly correspond to length of variable -- e.g. 'popno' is e.g. 1 while 'pop' could be 'M 50+', so i'd usually go for something like `for p,pop`, but obviously not a big deal, and i violate myself all the time with
`nthings = len(things)`